lib/twemproxy_exporter/counter.rb

```diff
@@ -0,0 +1,19 @@
+module TwemproxyExporter
+  class Counter
+    def initialize(registry, name, desc)
+      @counter = Prometheus::Client::Counter.new(name, desc)
+      registry.register(@counter)
+      @last = 0
+    end
+
+    def count(value, labels = {})
```

Comment:
Actually, I looked at the data we're collecting and this seems to be working fine as-is. I think I'll keep an eye on this for a little longer before recommending any changes.